RevenueCatUI/Templates/Components/CornerBorder.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CornerBorder.swift
+//
+//  Created by Josh Holtz on 9/30/24.
+
+import Foundation
+import SwiftUI
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct CornerBorderModifier: ViewModifier {
+
+    struct BorderInfo {
+
+        let color: Color
+        let width: CGFloat
+
+        init(color: Color, width: Double) {
+            self.color = color
+            self.width = width
+        }
+
+    }
+
+    struct RaidusInfo {
+
+        let topLeft: CGFloat?
+        let topRight: CGFloat?
+        let bottomLeft: CGFloat?
+        let bottomRight: CGFloat?
```

Comment:
How does this work with RTL languages? Not sure how SwiftUI handles that in general, but should we rename them like so? 

```diff
-        let topLeft: CGFloat?
-        let topRight: CGFloat?
-        let bottomLeft: CGFloat?
-        let bottomRight: CGFloat?
+        let topStart: CGFloat?
+        let topEnd: CGFloat?
+        let bottomStart: CGFloat?
+        let bottomEnd: CGFloat?
```

Reply:
I should put a todo in here but we would have to manually flip it in this case for RTL languages since there isn't native support for this. Thanks for catching!